first-react-app/src/app/App.tsx

```diff
@@ -0,0 +1,30 @@
+import "./App.css";
+import React from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Home from "../pages/Home";
+import NotFound from "../pages/NotFound";
+import About from "../pages/About";
+import Forms from "../pages/Forms";
+
+localStorage.setItem("searchValue", "");
+
+export function App() {
```

Comment:
you should use arrow function instead of regular function.

```suggestion
export const App = () => {
```